en/guide/overriding-express-api.md

```diff
@@ -61,7 +61,7 @@ Object.defineProperty(app.request, 'ip', {
 
 ## Prototype
 
-In order to provide the Express.js API, the request/response objects passed to Express.js (via `app(req, res)`, for example) need to inherit from the same prototype chain. By default this is `http.IncomingRequest.prototype` for the request and `http.ServerResponse.prototype` for the response.
+In order to provide the Express.js API, the request/response objects passed to Express.js (via `app(req, res)`, for example) need to inherit from the same prototype chain. By default, this is `http.IncomingRequest.prototype` for the request and `http.ServerResponse.prototype` for the response.
```

Comment:
Let's use "Express" everywhere rather than inconsistently using "Express", "Express.js", per the glossary entry.

Reply:
It makes sense.

Reply:
However, I just left them as it was, in external references like blog/article titles.

Reply:
That's OK.